source/applications/initializers.md

```diff
@@ -0,0 +1,100 @@
+Initializers provide an opportunity to configure your application as it boots.
+
+There are two types of initializers: application initializers and application
+instance initializers.
+
+Application initializers are run as your application boots, and provide the
+primary means to configure [dependency injections](./dependency-injection) in
+your application.
+
+Application instance initializers are run as an application instance is loaded.
+They provide a way to configure the initial state of your application, as well
+as to set up dependency injections that are local to the application instance
+(e.g. A/B testing confurations).
+
+Initializers can perform any configuration operations, as long as they are
+synchronous. The user experience is degraded by attempting to block the booting
```

Comment:
Hm. I 100% agree there are no cases for deferring loading in instanceInitializers and we should just say it isn't possible.

I'm less sure of initializers. In Torii we use defer in an initializer to keep the app from booting while we do some coordination with the parent window, and that seems legitimate to me. I'd like to just shuffle the wording to not imply that you *cannot* do anything async, but simply emphasize that you should use other patterns. ala:

> Instance Initializers can perform synchronous configuration operations. Asynchronous operations that are intended for the first render of the application should take place in routing hooks. Moving asynchronicity into routing allows for a better user experience including a rendered "loading" substate.